### DISCUSSION TITLE
Rescan files and force full update

### DISCUSSION THREAD

**Discussion by xbette** [pymedusa/Medusa#10785] [Ideas & Feature requests]
Hi.

could it be an idea to change how the "rescan files" and "force full update" works. 

Maybe more just the rescan files.. 

I think, if it just could scan the last season instead of EVERY single file then a scan would not take long to go through.. 

if you then want Medusa to scan every file you could use the "force full update" :D

**Comment by p0psicles** [comment]
That might work for your use case. But there will be definitely users who scan and expect it to scan all files.

**Reply by xbette** [reply]
Yeah okay, that makes sense

**Reply by xbette** [reply]
maybe you could make a new button besides the "rescan files" that says "rescan last season" :D? 

<img width="661" alt="Skærmbillede 2022-07-01 kl  13 45 40" src="https://user-images.githubusercontent.com/3235438/176888607-f66c59a8-ab40-4cce-90b9-39a24e194986.png">
